buildSrc/build.gradle

```diff
@@ -1,11 +1,11 @@
 buildscript {
 	repositories.jcenter()
 	dependencies {
-		classpath 'org.xtext:xtext-gradle-plugin:1.0.15'
+		classpath 'org.xtext:xtext-gradle-plugin:2.0.4'
 	}
 }
 
-ext.xtextVersion = '2.10.0'
+ext.xtextVersion = '2.17.0'
```

Comment:
2.18.0?

Reply:
can do. does not really matter, but when we are already on it...